client/blocks/reader-full-post/wp-iframe-resize.js

```diff
@@ -0,0 +1,86 @@
+/**
+ * This function is inspired by `wp-includes/js/wp-embed.js` of WP.org.
+ * It actually waits for a message from within the iFrame.
+ * The message will contain the actual height of the iFrame.
+ *
+ * @param {Element} contentWrapper The content wrapper element.
+ * @param {boolean} removeListener When true, remove the listener instead of adding it.
+ */
+const WPiFrameResize = ( contentWrapper, removeListener ) => {
+	const receiveEmbedMessage = function ( e ) {
+		const data = e.data;
+
+		if ( ! data ) {
+			return;
+		}
+
+		if ( ! ( data.secret || data.message || data.value ) ) {
+			return;
+		}
+
+		if ( /[^a-zA-Z0-9]/.test( data.secret ) ) {
+			return;
+		}
+
+		const iframes = contentWrapper.querySelectorAll( 'iframe[data-secret="' + data.secret + '"]' );
+		const blockquotes = document.querySelectorAll(
```

Comment:
This should also query only in `contentWrapper`.

Reply:
Good catch! 5e90b04